Microsoft.Toolkit.Uwp.Services/Core/ExtensionMethods.cs

```diff
@@ -31,7 +28,7 @@ public static class ExtensionMethods
         /// <returns>Returns string value.</returns>
         public static string ToSafeString(this object value)
```

Comment:
The warning should also be added on these methods since they are static extension methods